### ISSUE TITLE
Documentation needed

### ISSUE THREAD

**Issue by ghost** [marguerite/nodejs-packaging#26]
There should has a document to explain the exactly meaning of each macros, eg, `%nodejs_copy`, `%nodejs_prep`, because not all the people would like to learn Ruby just for building a package.

**Comment by guoyunhe** [comment]
Yes, indeed. Need to know how to generate source and spec files.
